docker-compose.yml

```diff
@@ -103,13 +103,16 @@ services:
     image: ghcr.io/celestiaorg/celestia-zkevm-ibc-demo/celestia-prover:latest
     container_name: celestia-prover
     environment:
-      - TENDERMINT_RPC_URL=http://celestia-network-validator:26657
-      - RPC_URL=http://reth:8545
```

Comment:
the RPC_URL doesn't need to be specified here b/c it already defaults to this value.

the TENDERMINT_RPC_URL needs to be overridden to use SimApp instead of Celestia.